app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+
+from app import errors
```

Comment:
Your code below will look cleaner if you explicitly import errors. This will improve readabllity and will remove necessity to write `error.ErrorName `